### PR TITLE
Bug 1080133 — Ignore empty mozlog type namespace.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by Natim** [comment]
I don't know what I forgot because it seems that the log is written to stdout even with the `logger.propagate = false`

File: test/schema.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var writer = {
   }
 };
 
-var logger = fxaLog('test.schema');
+var logger = fxaLog("test.schema");
```
Comment (seanmonstar) [inline test/schema.js]: Keep the single quotes.

File: test/schema.js [hunk 2]
```diff
@@ -43,6 +43,29 @@ describe('schema', function() {
     assert.equal(out.Type, 'test.schema.foo');
   });
 
+  describe("with emtpy namespace", function() {
```
Comment (seanmonstar) [inline test/schema.js]: Typo "emtpy", and use single quotes.

**Comment by seanmonstar** [comment]
With an empty name, you're receiving the app Logger, which mozlog configures to have a stdout handler. That's why it is printing to stdout.

**Comment by Natim** [comment]
Fixed

**Pull request merged**